index.rst

```diff
@@ -38,80 +31,63 @@ The Vera Rubin Observatory control system is highly distributed, containing mult
 These may require on-sky time for acquiring the data, and complex software to analyze and derive the configuration products.
 All of this data needs to be associated with this configuration in a way that is easily accessible and traceable.
 
-In this tech-note we discuss the basic operational procedures involved in handling CSC configuration.
+In this tech-note we discuss the basic organization of configurations and the operational procedures involved in handling CSC configuration.
 The document begins with the most simple case of bringing a CSC up with a specific configuration, and eventually details how to updating and testing newly defined configurations.
 
-For more technical details about how CSCs handle configuration, see `tstn-017 <https://tstn-017.lsst.io>`__.
-
-.. _section-configuration-interation:
-
-Configuration Interaction Use-Cases
-===================================
-
-Users will interact with configurations in multiple ways.
-In many cases, a user/operator will only need to change the configuration that is currently loaded and are not concerned with the contents of the configuration itself.
-This section illustrates example use-cases for these types of scenarios.
-
-Discovering available configurations
-------------------------------------
-
-The easiest way to get information from a CSC programmatically is by using a Jupyter notebook server.
-From a notebook, observers, developers and power users can easily interact with the system through Python.
-
-In order to know which configurations are available for a specific CSC, users can read the ``configurationsAvailable`` event.
-This is done using a ``salobj.Remote`` class.
-
-.. code-block:: python
-
-    from lsst.ts import salobj
-
-    domain = salobj.Domain()
-
-    atdome = salobj.Remote(domain, "ATDome")
+Most CSCs handle configuration in the same manner, however, there are a few exception to the rule, notably the Camera CSCs.
+These special cases are addressed in section-appendix-configuration-non-salObj_.
 
-    await atdome.start_task()
+For more technical details about how CSCs handle configuration, see `tstn-017 <https://tstn-017.lsst.io>`__.
 
-    config_available = await atdome.evt_configurationsAvailable.aget(timeout=5.)
+Configuration Repositories and Files
+====================================
 
-    # This will print the available labels.
-    print(config_available.labels)
+Because we need to be able to modify configurations without re-deploying components, configuration files are stored in their own repositories and kept separated from code.
+The configuration repository associated with each CSC is found in the configuration column of the `Master CSC Table`_.
+A single configuration repository often contains configuration files for multiple CSCs, each with it's own folder.
+Inside the a given CSC folder, the configuration files are then arranged into folders according to the version.
```

Comment:
"Inside the a  given..." -> "Inside the" or "Inside a"?

I think this needs some rephrasing. Maybe do an itemized description?


Something like:

```
Because we need to be able to modify configurations without re-deploying components, configuration files are stored in their own repositories and kept separated from code.
The configuration repository associated with each CSC is found in the configuration column of the `Master CSC Table`_.
These configuration repositories are organized as follows:

- A single repository may host configurations for multiple CSCs.
- CSC configurations are stored in folders with the CSC name.
- In each CSC folder there are sub-folders for different versions of the CSC configuration.
- A folder for a given version contains up to three different **types** of configuration files, all of which are yaml files.
  The types of configuration are:
  - Initial: `_init.yaml`
  - Site-specific: `_summit.yaml`, `_ncsa.yaml` ...
  - Configuration overrides.  
- The three different configuration files correspond to the levels and order in which they are read.
- Any parameter specified in a higher-level configuration file will override any value that is declared at a lower-level.
    #. ``_init.yaml`` This **required** file contains all values that are expected to be common to all sites and/or be relatively static in operations.
        This file may contain a complete set of parameters, but is only required to do so if no site-specific configuration file exists.

     #. ``_<site>.yaml`` This **optional** file contains contain site specific configuration parameters such as IP addresses and ports.
        Many CSCs have site specific files.
        Between this file and the ``_init.yaml`` file, **the configuration must be fully defined**

     #. ``filename.yaml`` - These **optional** files, referred to as configuration overrides, are only to be used when the values declared in the previous files require changes.
        These files are loaded manually by the users as is demonstrated in the section-configuration-interaction_.
```

Reply:
Included but I did tweak it a fair bit. So give it a once-over.